test/utils/parsers/get_applications_parser.go

```diff
@@ -19,19 +19,22 @@ func ParseJxGetApplications(s string) (map[string]Application, error) {
 	answer := make(map[string]Application, 0)
 	lines := strings.Split(strings.TrimSpace(s), "\n")
 	headerFound := false
+	fieldCount := 3
```

Comment:
So what's the deal with initialising this to 3, if you are going to set it to the field count in the header later?